enclave_manager/avalon_enclave_manager/singleton/enclave_manager.py

```diff
@@ -620,8 +561,10 @@ def main(args=None):
         logging.getLogger("STDERR"), logging.WARN)
 
     parse_command_line(config, remainder)
-    logger.info("Starting Enclave manager")
-    start_enclave_manager(config)
+    logger.info("Initialize enclave_manager")
```

Comment:
For debugging, maybe it would be better to print this is a `singleton` enclave manager.  Something like:

`logger.info("Initialize singleton enclave_manager")`